poetry/factory.py

```diff
@@ -153,3 +153,6 @@ def create_legacy_repository(
             cert=get_cert(auth_config, name),
             client_cert=get_client_cert(auth_config, name),
         )
+
+    def get_source_information(self, source_name):  # type: (str) -> Dict[str, str]
```

Comment:
Lets add a comment here so we know why this is empty for now.

Reply:
I removed it completely because it was not used (it was a leftover of previous changes).